contracts/games/gems/GemGame.sol

```diff
@@ -1,21 +1,63 @@
 // Copyright (c) Immutable Pty Ltd 2018 - 2024
 // SPDX-License-Identifier: Apache 2
+// solhint-disable not-rely-on-time
```

Comment:
we don't need this anymore

Reply:
we do, specifically for this line https://github.com/immutable/contracts/pull/200/files#diff-fe9f97ca149602fd40e7caaba1458383ddde181249e482aaa94bea8bbc412803R61

It's for any code using `block.timestamp`

Reply:
Ah!